patroni/dcs/raft.py

```diff
@@ -45,17 +49,17 @@ def resolve_host(self):
 
 class SyncObjUtility(object):
 
-    def __init__(self, otherNodes, conf, retry_timeout=10):
+    def __init__(self, otherNodes: Collection[Union[str, TCPNode]], conf: SyncObjConf, retry_timeout: int = 10) -> None:
         self._nodes = otherNodes
         self._utility = TcpUtility(conf.password, retry_timeout / max(1, len(otherNodes)))
 
-    def executeCommand(self, command):
+    def executeCommand(self, command: List[Any]) -> Any:
         try:
             return self._utility.executeCommand(self.__node, command)
         except Exception:
             return None
 
-    def getMembers(self):
+    def getMembers(self) -> Optional[List[str]]:
         for self.__node in self._nodes:
```

Comment:
My IDE doesn't like this. Setting an attr in a method outside of init and overwriting it in a for loop. Perhaps it should have a new object type for node/command. I know raft isn't important/maintained/supported, so will ignore.